cohort-engine/src/main/java/com/ibm/cohort/engine/cqfruler/MeasureSupplementalDataEvaluation.java

```diff
@@ -63,15 +65,19 @@ public static void populateSDEAccumulators(Context context, Patient patient,
 						code = ((Code) sdeListItem).getCode();
 						break;
 					case "ArrayList":
-						if (!((ArrayList<?>) sdeListItem).isEmpty()) {
+						if (((ArrayList<?>) sdeListItem).isEmpty()) {
+							return;
+						}
+						else {
 							code = ((Coding) ((ArrayList<?>) sdeListItem).get(0)).getCode();
-						} else {
-							continue;
 						}
 						break;
+					default: 
+						throw new UnsupportedOperationException("Supplemental data evaluation not supported for type: " + sdeListItem.getClass());
 					}
+					
 					if (null == code) {
-						continue;
+						return;
```

Comment:
I have the same concern here as Matt's above comment regarding the switch from `continue` to `return`.

Reply:
Good catch, realistically I'm not convinced we should hit that case anyway (thanks cqf-ruler for 0 tests and examples) but I rejiggered the code.